test-data/unit/cmdline.test

```diff
@@ -476,6 +476,50 @@ ignore_missing_imports = True
 [out]
 main.py:2: note: Revealed type is 'Any'
 
+[case testFailedImportCanBeFoundInParent]
+# cmd: mypy main.py
+[file main/subdir/main.py]
+import wrongdir
+[file main/subdir/__init__.py]
+[file main/wrongdir.py]
+[file main/__init__.py]
+[out]
+main.py:1: error: Cannot find implementation or library stub for module named 'wrongdir'
+main.py:1: note: You may be running mypy in the wrong directory, the above module can be imported by moving to the project root.
+
+[case testFailedImportCanBeFoundInParentsParent]
+# cmd: mypy main.py
+[file main/subdir1/subdir2/main.py]
+import wrongdir
+[file main/subdir1/subdir2/__init__.py]
+[file main/subdir1/__init__.py]
+[file main/wrongdir.py]
+[file main/__init__.py]
+[out]
+main.py:1: error: Cannot find implementation or library stub for module named 'wrongdir'
+main.py:1: note: You may be running mypy in the wrong directory, the above module can be imported by moving to the project root.
+
+[case testFailedImportCannotBeFoundInParent]
+# cmd: mypy main.py
+[file main/subdir/main.py]
+import missing
+[file main/subdir/__init__.py]
+[file main/wrongdir.py]
+[file main/__init__.py]
+[out]
+main.py:1: error: Cannot find implementation or library stub for module named 'missing'
+main.py:1: note: See https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports
+
+[case testImportInParentButNoInit]
+# cmd: mypy main.py
+[file main/subdir/main.py]
+import wrongdir
+[file main/wrongdir.py]
+[file main/__init__.py]
+[out]
+main.py:1: error: Cannot find implementation or library stub for module named 'wrongdir'
+main.py:1: note: See https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports
```

Comment:
Command line tests are quite expensive to run. Could you reduce the number of test cases to 2, for example? Maybe you can combine tested features.